xml/FrameworksIndex/netframework-3.0.xml

```diff
@@ -119774,6 +119774,10 @@
       <Member Id="M:System.Windows.Xps.XpsDocumentWriter.CancelAsync" />
       <Member Id="M:System.Windows.Xps.XpsDocumentWriter.CreateVisualsCollator" />
       <Member Id="M:System.Windows.Xps.XpsDocumentWriter.CreateVisualsCollator(System.Printing.PrintTicket,System.Printing.PrintTicket)" />
+      <Member Id="M:System.Windows.Xps.XpsDocumentWriter.raise_WritingCancelled(System.Object,System.Windows.Documents.Serialization.WritingCancelledEventArgs)" />
```

Comment:
@joelmartinez this seems like a new issue with mdoc (I'm still on the previous version). The event should be documented (https://docs.microsoft.com/en-us/dotnet/api/system.windows.xps.xpsdocumentwriter.writingcancelled?view=netframework-4.7.2) but not the internal implementation.

Reply:
Hmm, ok … can you make a GH issue for this one? also, where can I find the assemblies that went into this change in the binaries repo, the `mdoc_update` branch ?

Reply:
Yes. I've fixed the dependencies issues last night.

Reply:
And sure, I'll create the bug.

Reply:
So strange … mdoc explicitly filters out methods that start with `raise_` here:  
https://github.com/mono/api-doc-tools/blob/5d1f8a2044f0fd15946ceb05f1a5d850e83822b7/mdoc/Mono.Documentation/Updater/DocUtils.cs#L432

And the metadata for that method on netfx 3 shows that it has the `IsSpecialName` flag … so it definitely should have been filtered.

Reply:
Wow! Very odd indeed.